src/core/rubberbandmodel.h

```diff
@@ -111,11 +119,12 @@ class RubberbandModel : public QObject
     void crsChanged();
     //! \copydoc frozen
     void frozenChanged();
-
+    void currentPositionTimestampChanged();
```

Comment:
why is this saved in the rubber band? it doesn't sounds like the right place for this.

Reply:
We could save it in the tracker session as well. 
But because the current coordinate (and with this all the "recording" data) are stored in the rubberband, I decided to put it into rubberband. Yet, the timestamp is only stored in the m-coordinate on tracking, but maybe in future, we use it on other places as well. So it won't be nice to have it directly in the tracking session IMHO.

Reply:
ah I see, thanks for the explanation.
Do you think it might be worth decoupling the 2 in the future (rubberband and location) or does it make sense to keep them coupled?

Reply:
I think the rubberband model and the track are both a list of vertices on which we work, which we digitalize, which are converted/applied to geometries.

I think it makes sense to keep these two coupled. Probably treating the topic in here as "m value" rather than "positiontimestamp"?

Reply:
> Probably treating the topic in here as "m value" rather than "positiontimestamp"?

Yes, that would maybe make it more clear.

Reply:
yes, sounds perfect to me too. That would make it compatible (API wise) with external sensors too.